README.md

```diff
@@ -1,124 +1,10 @@
-    -v GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL:$GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL \
-    -v OAUTH_TOKEN="$(printf $OAUTH_TOKEN | base64 -w 0)" \
-    -v GKE_AUTH="$(cat $AUTH_FILE | base64 -w 0)" \
-    -v GITHUB_ORG:$GITHUB_ORG -v GITHUB_REPO:$GITHUB_REPO \
-    -f manifests/cluster-infra
-```
-> Note: Use `-v GKE_AUTH="$(echo $AUTH_FILE | base64 -w 0)"` if you're passing the data directly into `$AUTH_FILE`
-- The output will show the ingress IP which will be used to point the domain name to. Alternatively you can see it from the GKE/Services tab.
-- Set the `A record` for `<DOMAIN_NAME>` to point to `nginx-ingress-controller` IP address.
-- The services will be accessible at:
-  * Grafana :: `http://<DOMAIN_NAME>/grafana`
-  * Prometheus ::  `http://<DOMAIN_NAME>/prometheus-meta`
-
-### Setup GitHub Actions
-Place a workflow file in the `.github` directory of the repository.
-See the [prometheus/prometheus](https://github.com/prometheus/prometheus) repository for an example.
-
-Create a github action `AUTH_FILE` secret with the base64 encoded content of the `service-account.json` file.
-```
-cat $AUTH_FILE | base64 -w 0
-```
-
-## Usage
-### Trigger tests via a Github comment.
----
-> Due to the high cost of each test, only maintainers can manage tests.
-
-Starting:
-- `/benchmark` - benchmark PR with the master branch.
-- `/benchmark master` - same as above
-- `/benchmark v2.4.0` - can use any release version
-
-Restarting:
-- Comment `/benchmark` again on the PR. If the hash of the last commit has changed, the test will be restarted.
-- To restart the test with a different Prometheus release version comment `/benchmark <new_rel_version>`
-
-Stopping:
-- Comment `/benchmark cancel`.
-
-### Start a benchmarking test manually
----
-
-- Set the following environment variables.
-```
-export RELEASE=<master or any prometheus release(ex: v2.3.0) >
-export PR_NUMBER=<PR to benchmark against the selected $RELEASE>
-```
-
-- Create the nodepools for the k8s objects
-```
-./prombench gke nodepool create -a $AUTH_FILE \
-    -v ZONE:$ZONE -v PROJECT_ID:$PROJECT_ID -v CLUSTER_NAME:$CLUSTER_NAME \
-    -v PR_NUMBER:$PR_NUMBER -f manifests/prombench/nodepools.yaml
-```
-
-- Deploy the k8s objects
-```
-./prombench gke resource apply -a $AUTH_FILE \
-    -v ZONE:$ZONE -v PROJECT_ID:$PROJECT_ID -v CLUSTER_NAME:$CLUSTER_NAME \
-    -v PR_NUMBER:$PR_NUMBER -v RELEASE:$RELEASE -v DOMAIN_NAME:$DOMAIN_NAME \
-    -f manifests/prombench/benchmark
-```
-
-## Buliding from source
-To build Prombench and related tools from source you need to have a working Go environment with go modules enabled. Prombench uses [promu](https://github.com/prometheus/promu) to build the binaries.
-```
-make build
-```
+## Funcbench
+Benchmarking of the time series database that Prometheus uses by default. The benchmark is initiated by commenting on a pull request in the Prometheus/TSDB repository. See Funcbench [README.md](/cmd/funcbench/README.md) for more information.
```

Comment:
We can benchmark any bench functions right? not just the tsdb functions.
```suggestion
Benchmarking of a given function using the built in Golang benchmarking tooling. The benchmark is initiated by commenting on a pull request. See Funcbench [README.md](/cmd/funcbench/README.md) for more information.
```

Reply:
True